core/include/drivers/stm32_risaf.h

```diff
@@ -0,0 +1,22 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2021-2024, STMicroelectronics
+ */
+
+#ifndef __DRIVERS_STM32_RISAF_H__
+#define __DRIVERS_STM32_RISAF_H__
+
+/*
+ * stm32_risaf_clear_illegal_access_flags() - Clears flags raised when an
+ * illegal access occurs on a memory region
+ */
+void stm32_risaf_clear_illegal_access_flags(void);
+
+/*
+ * stm32_risaf_dump_erroneous_data() - Dumps the data associated to an illegal
```

Comment:
nitpikcing: prefer `Print` instead of `Dump`. Maybe the function label should be changed also.
Could say that the function is stubbed when `CFG_TEE_CORE_DEBUG` is disabled.